src/bot.ts

```diff
@@ -1,68 +1,63 @@
-  const vouchRepository = connection.getCustomRepository(VouchRepository);
-
-  const client = new Client();
+  await dbDataSource.initialize();
+  const discordClient = new DiscordClient();
   const eventEmitter = new EventEmitter();
 
-  const checkVouchCommandHandler = new CheckVouchCommandHandler(client, vouchRepository);
-  const plusVouchCommandHandler = new PlusVouchCommandHandler(client, vouchRepository, eventEmitter);
-  const minusVouchCommandHandler = new MinusVouchCommandHandler(client, vouchRepository, eventEmitter);
+  const checkVouchCommandHandler = new CheckVouchCommandHandler(discordClient, vouchRepository);
+  const plusVouchCommandHandler = new PlusVouchCommandHandler(discordClient, vouchRepository, eventEmitter);
+  const minusVouchCommandHandler = new MinusVouchCommandHandler(discordClient, vouchRepository, eventEmitter);
   const deleteVouchEvent = new DeleteVouchEvent(vouchRepository, eventEmitter);
 
   setupDiscordEventsAndHandlers(
-    client,
+    discordClient,
     vouchRepository,
     minusVouchCommandHandler,
     plusVouchCommandHandler,
     checkVouchCommandHandler,
-    deleteVouchEvent
+    deleteVouchEvent,
   );
 
-  void client.login(process.env.DISCORD_TOKEN);
+  void discordClient.login(process.env.DISCORD_TOKEN);
 }
 
 function setupDiscordEventsAndHandlers(
-  client: Client,
+  discordClient: DiscordClient,
   vouchRepository: VouchRepository,
   minusVouchCommandHandler: MinusVouchCommandHandler,
   plusVouchCommandHandler: PlusVouchCommandHandler,
   checkVouchCommandHandler: CheckVouchCommandHandler,
-  deleteVouchEvent: DeleteVouchEvent
+  deleteVouchEvent: DeleteVouchEvent,
 ) {
-  client.once('ready', () => {
-    setStatus(client.user);
-    void parseOfflineMessages(vouchRepository, client, minusVouchCommandHandler, plusVouchCommandHandler);
+  discordClient.once('ready', () => {
+    setStatus(discordClient.user);
+    void parseOfflineMessages(vouchRepository, discordClient, minusVouchCommandHandler, plusVouchCommandHandler);
   });
 
-  client.on('message', (message: Message) => void plusVouchCommandHandler.handle(message));
-  client.on('message', (message: Message) => void minusVouchCommandHandler.handle(message));
-  client.on('message', (message: Message) => void checkVouchCommandHandler.handle(message));
+  discordClient.on('message', (message: Message) => void plusVouchCommandHandler.handle(message));
+  discordClient.on('message', (message: Message) => void minusVouchCommandHandler.handle(message));
+  discordClient.on('message', (message: Message) => void checkVouchCommandHandler.handle(message));
 
-  client.on('messageDelete', (message: Message | PartialMessage) => void deleteVouchEvent.handle(message));
+  discordClient.on('messageDelete', (message: Message | PartialMessage) => void deleteVouchEvent.handle(message));
 }
 
 function ensureEnvironmentVariablesAreSet() {
   const requiredEnvVars = [
     'DISCORD_TOKEN',
-    'PREFIX',
```

Comment:
question: As far as you can tell, this isn't used anywhere?

Reply:
Looking back through the history, I found that in the begin this was kind of the Prefix for the commands. It was removed in a certain point with a code refactor and it's not used anymore.